submissions/description/unvalidated_redirects_and_forwards/open_redirect/get_based/template.md

```diff
@@ -12,9 +12,9 @@ GET-based open redirects can result in reputational damage for the business as c
 
 ## Steps to Reproduce
 
-1. Using a browser, navigate to: {{URL}}
-1. Copy and modify the URI so that the URL redirects to `url=<http://www.bugcrowd.com/`
-1. Submit this in a new browser window and you should be redirected to the Bugcrowd website.
+1. Using a browser, navigate to the open redirect on the endpoint: {{URL_with_no_redirect}}
```

Comment:
Oh, perhaps better wording here is "navigate to the vulnerable endpoint:"

'the open redirect on the endpoint:' is strange wording here considering that it does not include the redirect code at this point.